tests/methods/test_maskers.py

```diff
@@ -10,37 +12,78 @@ def test_mask_has_correct_shape_univariate():
     input_data = _get_univariate_input_data()
     number_of_masks = 5
 
-    result = _call_masking_function(input_data, number_of_masks=number_of_masks)
+    result = _call_masking_function(input_data,
+                                    number_of_masks=number_of_masks)
 
     assert result.shape == tuple([number_of_masks] + list(input_data.shape))
 
 
+def test_mask_has_correct_type_univariate():
+    """Test masked data has the correct dtype for a univariate input."""
+    input_data = _get_univariate_input_data()
+    number_of_masks = 5
+
+    result = generate_masks(input_data, number_of_masks=number_of_masks)
+
+    assert result.dtype == bool
+
+
+def test_generate_time_step_masks_dtype_multivariate():
+    """Test masked data has the correct dtype for a multivariate input."""
+    input_data = _get_multivariate_input_data()
```

Comment:
I couldn't understand what is meant by `multivariate_input_data`. Does it mean 2D data or more data with more than two dimensions? Currently, the function `_get_multivariate_input_data` only generates 2D data. When I changed it to 3D data, the tests failed.

Reply:
I meant multivariate time series. That is still 2D because we have time steps * channels. 
I changed the function title from ```_get_multivariate_input_data``` to ```_get_multivariate_time_series``` to make it clear that it only gets time series data. 
Let me know if you think I should clarify this some more.